galaxy.yml

```diff
@@ -0,0 +1,70 @@
+---
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: "networktocode"
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: "netauto"
+
+# The version of the collection. Must be compatible with semantic versioning
+version: "1.0.0"
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: "README.md"
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+  - "Network to Code <opensource@networktocode.com>"
+
+
+### OPTIONAL but strongly recommended
+# A short summary description of the collection
+description: "Multi-vendor Ansible Collection for Network Automation"
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+# license:
+# - GPL-2.0-or-later
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
+license_file: "LICENSE"
+
+# A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+# requirements as 'namespace' and 'name'
+tags:
+  - "networking"
+  - "network automation"
+  - "multi-vendor"
+
+# Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+# collection label 'namespace.name'. The value is a version range
+# L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
+# range specifiers can be set and are separated by ','
+dependencies: {}
+
+# The URL of the originating SCM repository
+repository: "https://github.com/networktocode/ntc-ansible"
+
+# The URL to any online docs
+documentation: "https://github.com/networktocode/ntc-ansible"
+
+# The URL to the homepage of the collection/project
+homepage: "https://github.com/networktocode/ntc-ansible"
+
+# The URL to the collection issue tracker
+issues: "https://github.com/networktocode/ntc-ansible/issues"
+
+# A list of file glob-like patterns used to filter any files or directories that should not be included in the build
+# artifact. A pattern is matched from the relative path of the file or directory of the collection directory. This
+# uses 'fnmatch' to match the files or directories. Some directories and files like 'galaxy.yml', '*.pyc', '*.retry',
+# and '.git' are always filtered
+build_ignore:
+  - ".pytest_cache"
+  - ".vscode"
+  - "*.tar.gz"
+  - "poetry.lock"
+  - "pyproject.toml"
```

Comment:
You should include the pyproject.toml in the collection build. Also include a requirements.txt for environments without poetry:

`poetry export -f requirements.txt --output requirements.txt`